src/Dev-9/FindValueOfKeyInSection.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Class for search value of key in section
+ */
+public class FindValueOfKeyInSection {
+    /**
+     * Find value
+     *
+     * @param A List of lines
+     * @return value of key
+     */
+    public int findValue(String[] A) {
+        DataInput dataInput = new DataInput();
+        int value = 1;
+        if (findSection(A)) {
+            for (int i = 0; i < A.length; i++) {
+                if (A[i].contains(dataInput.getKey())) {
+                    String[] v = A[i].split("=");
+                    System.out.println(v[1]);
+                }
+            }
+            return value;
+        }
+        return 0;
+    }
+
+    /**
+     * @param A List of arguments file
+     */
+    private boolean findSection(String[] A) {
+        DataInput dataInput = new DataInput();
+        for (int i = 0; i < A.length; i++) {
+            if (A[i].contains(dataInput.getSection())) {
```

Comment:
Here dataInput do not contain section, the method inputSection was no called, this is bug